fideslog/sdk/python/client.py

```diff
@@ -1,6 +1,8 @@
 # pylint: disable=too-many-arguments
```

Comment:
Also this gets rid of the pylint info-level message due to the conditional `import`:
```suggestion
# pylint: disable=import-outside-toplevel, too-many-arguments
```